how-we-work/donut.html.md

```diff
@@ -5,14 +5,15 @@ meta_description: >
   teammates. Learn how we use a Slack bot to help with that!
 ---
 
-Periodically during the week, [Donut](https://donut.com) will pair people in the #caffe channel. It
+Two times per week, [Donut](https://donut.com) will pair 3 people in the #coffee channel. It
```

Comment:
The text here was generic for a reason... so that we could change the donut without having to change the text.

I would rather have the more generic text as it's easier to manage.

Reply:
ok @mtylty it makes sense